types/node/globals.d.ts

```diff
@@ -130,10 +130,10 @@ interface ImportMeta {
 }
 
 /*-----------------------------------------------*
- *                                               *
- *                   GLOBAL                      *
- *                                               *
- ------------------------------------------------*/
+  *                                               *
```

Comment:
I didn't do any deliberate formatting, this is my VSCode (which It must be following the prettier config on the root). Let me know if that's not the case.

Reply:
You can run a check like `prettier --check [path]` to verify if this current format to make sure you're working with the right format.
But I verify already, and it's fine!